styles/styles.css

```diff
@@ -0,0 +1,142 @@
+* {
+  font-family: 'Nanum Gothic', sans-serif;
+}
+
+li {
+  list-style-type: none
+}
+
+ul {
+  padding-inline-start: 0px;
+}
+
+h2 {
+  text-align: center
+}
+
+a:hover {
+  background-color: black
+}
+
+a:link {
+  color: grey;
+  text-decoration: none;
+}
+
+a:visited {
+  color: grey;
+  text-decoration: grey;
+}
+
+.navbar {
+  width: 100vw;
+  margin: 3rem;
+  display: grid;
+  grid-template: 1fr/ 1fr 1fr 1fr 1fr 1fr;
+  font-weight: bold;
+  text-align: left;
+} 
+
+
+#startly {
+  grid-column-start: 1;
+  grid-column-end: span 3;
+  text-align: left;
+}
+
+.title{
+  display: grid;
+  grid-template: 1fr 1fr 1fr/ 1fr;
+  margin-top: 7rem
+}
+
+.title div {
+  text-align: center;
+}
+
+
+.title div:nth-of-type(3) {
+  border-radius: 10px;
+  border: 2px solid grey;  
+  text-align: center;
+  padding-top: 1.5rem;
+  margin-left: 27rem;
+  margin-right: 27rem;
+}
+
+.quote {
+  border-bottom: 2px solid grey;
+  border-top: 2px solid grey;
+  margin-top: 5rem;
+  display: grid;
+  grid-template: 1fr 1fr/ 1fr;
+  padding: 4rem;
+}
+
+blockquote, blockquote + p {
+  font-style: italic;
+}
+
+blockquote + p {
+  text-align: right
+}
+
+
+
+.team {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-content: space-between;
+}
+
+.team li {
+  margin: 6rem;
+}
+
+.team li div {
+  text-align: center
+}
+
+.head {
+  width: 100%;
+  height: 7rem;
+  margin-top: 5rem
+}
+
+.team img {
+  border-radius: 50%;
+  border: 0.1rem solid grey;
+  padding: .2rem
+}
+
+
+.foot {
+  width: 100vw;
+  margin: auto;
+  padding: 2rem;
+  border-top: 2px solid grey;
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr 1fr;
+  background-color: lightgray
+}
+
+
+
+
+.why {
+  width: 100vw;
+  margin: auto;
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr
+}
```

Comment:
You can try adding a grid inside of a grid that may help you align your text here.